changes-set.txt

```diff
@@ -101,6 +101,8 @@ make a github issue.)
 
 DONE:
 Date      Old       New         Notes
+22-Oct-23 spimeh    spimew      show relation to spimw
+22-Oct-23 spimev    spimevv     same as with spv/spvv
```

Comment:
I see no "spimevv" in the diffs ?
Did you mean to write the change for "speiv" ? In that case, I think the new label should be "speivw" for "weak form of speiv", since there is no additional DV condition.

Reply:
That's right

Reply:
~spvv has that label because it adds a DV condition to ~spv, which is not the case of ~speivv with respect to ~speiv. This is why I think ~speivw is a better label.